src/components/Wishlist.js

```diff
@@ -1,16 +1,16 @@
 import React, { Component } from "react";
 import { firebase } from "../firebase";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 export class Wishlist extends Component {
   constructor() {
```

Comment:
Instead of using constructor just define state directly in the class:

```
state = {
      text: "",
      url: "",
      textarea: "",
      items: []
    };	
```